cosmo_tester/framework/test_hosts.py

```diff
@@ -1128,6 +1210,15 @@ def destroy(self, passed=None):
             self._infra_client.tenants.delete(self.tenant)
             self.tenant = None
 
+    def dump_xfs(self):
+        for instance in self.instances:
+            instance.prepare_and_dump_xfs()
+            self._logger.info('Waiting for instance %s to dump XFS',
```

Comment:
Probably good to log when we finish waiting for each instance as well in the next loop.

Reply:
already logging success/failure inside is_complete, which is equivalent